api/routes/team.js

```diff
@@ -21,8 +21,10 @@ router.get('/:players/:salaryCap', async (req, res) => {
       let statsJSON = await Stats.findOne({ Rk: rank }).exec();
       statsJSONs.push(statsJSON);
       let playerJSON = await Players.findOne({ Rk: rank }).exec();
-      const salary = parseInt(playerJSON.salary.substring(1));
-      totalSalary += salary;
+      if (playerJSON.salary) {
```

Comment:
Does this return an error if the salary property is not found? If so, maybe add a hasOwnProperty() check.

Reply:
nope! it just returns undefined